test/e2e/create_test.go

```diff
@@ -53,17 +53,35 @@ var _ = Describe("Podman create", func() {
 	})
 
 	It("podman create container based on a remote image", func() {
-		session := podmanTest.Podman([]string{"create", BB_GLIBC, "ls"})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(0))
+		// Changing session -> csession to avoid conflict for 2 session variable
+		csession := podmanTest.Podman([]string{"create", BB_GLIBC, "ls"})
```

Comment:
I think we can leave this as `session` - I am not following the reason for the variable name change here.

Reply:
Actually, there was more than 1 check included in that `It` block. So I had changed the variable name.
Reverted it now.